tox.ini

```diff
@@ -15,75 +12,70 @@ src_path = {toxinidir}/src/
 tst_path = {toxinidir}/tests/
 
 [testenv]
-passenv =
-    PYTHONPATH
-    CHARM_BUILD_DIR
-    MODEL_SETTINGS
-    KUBECONFIG
-setenv =
-    PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
-    PYTHONBREAKPOINT=ipdb.set_trace
-    PY_COLORS=1
+passenv = 
+	PYTHONPATH
+	CHARM_BUILD_DIR
+	MODEL_SETTINGS
+	KUBECONFIG
+setenv = 
+	PYTHONPATH = {toxinidir}:{toxinidir}/lib:{[vars]src_path}
+	PYTHONBREAKPOINT=ipdb.set_trace
+	PY_COLORS=1
 
 [testenv:update-requirements]
-allowlist_externals =
-    bash
-    find
-    pip-compile
-    xargs
+allowlist_externals = 
+	bash
+	find
+	pip-compile
+	xargs
 commands = 
-    ; we must preserve the order of compilation, since each *.in file depends on some *.txt file.
-    ; For example, requirements-unit.in depends on requirements.txt and we must compile first
-    ; requirements.txt to ensure that requirements-unit.txt get the same dependency as the requirements.txt
```

Comment:
Do we need to remove comments (including Copyright statement)? I 'm afraid we could remove some useful information from tox files.

Reply:
yes, that's a good point. Let me try to find a way to improve the reformatting

Reply:
managed to get it working, but required a bit of custom implementation that I took from [here](https://gist.github.com/Jip-Hop/d82781da424724b4018bdfc5a2f1318b). See the new commit [here](https://github.com/deusebio/kubeflow-ci-cli/pull/2/commits/91199cb5aaeebaf6f222d9b59e7348e1728c992c)

Reply:
Awesome work!